src/client/components/ShoppingCart/ModifyProductQuantity.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './ShoppingCart.styles.css';
+
+function ModifyProductQuantity(props) {
+  const { product, shoppingCart } = props;
+
+  const [counter, setCounter] = useState(product.quantity);
+
+  const incrementCounter = () => {
+    setCounter(counter + 1);
+
+    // console.log('in increment cart');
+    const result = shoppingCart.find(({ id }) => id === product.id);
+    result.quantity += 1;
+    // console.log(shoppingCart);
+  };
+
```

Comment:
Also can you remove all the many coments with console.log 👍

Reply:
done